t/007-decryptCookieV3.t

```diff
@@ -74,8 +74,8 @@ Process a valid V1 cookie
     	    pxconfig.cookie_secret = "perimeterx"
     	    pxconfig.px_debug = true
     	    pxconfig.block_enabled = true
+    	    pxconfig.send_page_requested_activity = false
```

Comment:
on all modules, we should align that sending activities will be set to true by defaults (block and page_requested)